.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
     - 3.5
     - 3.6
     - 3.7
+    - 3.8-dev
```

Comment:
The [`PROGRAMMING LANGUAGE :: PYTHON :: 3.8` classifier](https://pypi.org/search/?c=Programming+Language+%3A%3A+Python+%3A%3A+3.8) is now available. Should it be added to the local list?

What about a change note?